classes/PaynowAdminFormHelper.php

```diff
@@ -435,7 +453,8 @@ private function getConfigFieldsValues(): array
             'PAYNOW_PAYMENT_VALIDITY_TIME' => Configuration::get('PAYNOW_PAYMENT_VALIDITY_TIME'),
             'PAYNOW_ORDER_ABANDONED_STATE' => Configuration::get('PAYNOW_ORDER_ABANDONED_STATE'),
             'PAYNOW_ORDER_EXPIRED_STATE' => Configuration::get('PAYNOW_ORDER_EXPIRED_STATE'),
-            'PAYNOW_CREATE_ORDER_STATE' => Configuration::get('PAYNOW_CREATE_ORDER_STATE')
+            'PAYNOW_CREATE_ORDER_STATE' => Configuration::get('PAYNOW_CREATE_ORDER_STATE'),
+            'PAYNOW_RETRY_PAYMENT_BUTTON' => Configuration::get('PAYNOW_RETRY_PAYMENT_BUTTON'),
```

Comment:
jeśli coś służy to zmiany stanu włącz/wyłącz dodałbym suffix _ENABLED jak wyżej

Reply:
poprawione